### PR TITLE
feat(support bundle): chart, YAML update

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/2759

### CODE REVIEW

File: chart/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -11,3 +11,17 @@ subjects:
...
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: longhorn-support-bundle
+  labels: {{- include "longhorn.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (innobead) [inline chart/templates/clusterrolebinding.yaml:23]: Need to mention this in the doc.

File: chart/values.yaml [hunk 2]
```diff
@@ -41,6 +41,9 @@ image:
     backingImageManager:
       repository: longhornio/backing-image-manager
       tag: v3_20220808
+    supportBundleKit:
+      repository: rancher/support-bundle-kit
+      tag: v0.0.13
```
Comment (innobead) [inline chart/values.yaml:46]: `longhornio/support-bundle-kit` instead

File: chart/questions.yaml [hunk 3]
```diff
@@ -354,6 +354,12 @@ Disabling **Backupstore Poll Interval** also means to disable failed backup auto
     type: int
     min: 0
     default: 1
+  - variable: defaultSettings.supportBundleFailedHistoryLimit
```
Comment (innobead) [inline chart/questions.yaml:357]: Need the description.

**Review by innobead — CHANGES_REQUESTED** [review]

**Review by innobead — APPROVED** [review]

File: chart/templates/clusterrolebinding.yaml (hunk 1 above)
Comment (c3y1huang) [reply chart/templates/clusterrolebinding.yaml:23]: Update to https://github.com/longhorn/website/pull/620.

File: chart/values.yaml (hunk 2 above)
Comment (innobead) [reply chart/values.yaml:46]: Will need another new release, then update here before merging

**Pull request merged**